packages/flutter/lib/src/cupertino/app.dart

```diff
@@ -37,6 +37,8 @@ import 'theme.dart';
 /// This widget also configures the observer of the top-level [Navigator] (if
 /// any) to perform [Hero] animations.
 ///
+/// [CupertinoApp] isn't necessary for an iOS app.
```

Comment:
This needs more details and a proposed solution. 

```suggestion
/// The [CupertinoApp] widget isn't a required ancestor for other Cupertino 
/// widgets, but many Cupertino widgets could depend on the [CupertinoTheme]
/// widget, which the [CupertinoApp] composes. If you use Material widgets, a
/// [MaterialApp] also creates the needed dependencies for Cupertino widgets.
```

Reply:
Ah, Awesome
You are right. we need more details
let me add 
Thanks